[Bug] [icla plugin] "make dev" complie error

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.


### What happened

when I run "make dev", catch a compile error:

$ make dev
Building plugin ae to bin/plugins/ae/ae.so
Building plugin dbt to bin/plugins/dbt/dbt.so
Building plugin feishu to bin/plugins/feishu/feishu.so
Building plugin gitextractor to bin/plugins/gitextractor/gitextractor.so
Building plugin github to bin/plugins/github/github.so
Building plugin gitlab to bin/plugins/gitlab/gitlab.so
Building plugin icla to bin/plugins/icla/icla.so
command-line-arguments
plugins/icla/plugin_main.go:98:56: too many arguments in call to runner.DirectRun
	have (*cobra.Command, []string, Icla, []string, map[string]interface{})
	want (*cobra.Command, []string, core.PluginTask, map[string]interface{})
make: *** [Makefile:25: build-plugin] Error 2


### What you expected to happen

compile succeed, and run "go run main.go" to start an application.

### How to reproduce

use main branch , commit id is 0d3aa41e5e3db60e7ddfd3c3835f40e8c1c8eeb4. then run"make dev" command

### Anything else

check source code  plugins/icla/plugin_main.go, find the wrong code:
```golang
func main() {
    cmd := &cobra.Command{Use: "icla"}

    // TODO add your cmd flag if necessary
    // yourFlag := cmd.Flags().IntP("yourFlag", "y", 8, "TODO add description here")
    // _ = cmd.MarkFlagRequired("yourFlag")

    cmd.Run = func(cmd *cobra.Command, args []string) {
        runner.DirectRun(cmd, args, PluginEntry, []string{}, map[string]interface{}{          //  wrong code                                                                                     
            // TODO add more custom params here
        })  
    }   
    runner.RunCmd(cmd)
}
```
change  "runner.DirectRun(cmd, args, PluginEntry, []string{}, map[string]interface{}" to \
"runner.DirectRun(cmd, args, PluginEntry, map[string]interface{}" 
will work.

### Version

main

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/generator/template/plugin/plugin_main.go-template
+++ b/generator/template/plugin/plugin_main.go-template
@@ -97,7 +97,7 @@ func main() {
 	// _ = cmd.MarkFlagRequired("yourFlag")
 
 	cmd.Run = func(cmd *cobra.Command, args []string) {
-		runner.DirectRun(cmd, args, PluginEntry, []string{}, map[string]interface{}{
+		runner.DirectRun(cmd, args, PluginEntry, map[string]interface{}{
 			// TODO add more custom params here
 		})
 	}

--- a/plugins/icla/plugin_main.go
+++ b/plugins/icla/plugin_main.go
@@ -95,7 +95,7 @@ func main() {
 	// _ = cmd.MarkFlagRequired("yourFlag")
 
 	cmd.Run = func(cmd *cobra.Command, args []string) {
-		runner.DirectRun(cmd, args, PluginEntry, []string{}, map[string]interface{}{
+		runner.DirectRun(cmd, args, PluginEntry, map[string]interface{}{
 			// TODO add more custom params here
 		})
 	}

